src/Pos/Client/Txp/History.hs

```diff
@@ -120,10 +122,11 @@ getRelatedTxs addr txs = fmap DL.toList $
     handleRelatedTx (isOutgoing, isToItself) (tx, txId, dist) = do
         applyTxToUtxo (WithHash tx txId) dist
         ether $ identity %= filterUtxoByAddr addr
+        inputs <- (map toaOut . catMaybes) <$> mapM utxoGet (toList $ _txInputs tx)
```

Comment:
> Why can't I replace (toList $ _txInputs tx) with $ toList $ _txInputs tx? This results in a complicated type error.
because `$` has the lowest priority and that will change the semantics

> How can I rewrite (map toaOut . catMaybes) without brackets?
just remove brackets. This should work:
```
         inputs <- map toaOut . catMaybes <$> mapM utxoGet (toList $ _txInputs tx) 
```